examples/.gitkeep

```diff
@@ -1 +0,0 @@
-
```

Comment:
think we might need the `.gitkeep` file in order for the directory to remain in the repo since it has no files/subdirectories.

Reply:
The file is not being deleted, what I did was delete an unnecessary line break, as it had two line breaks.

Reply:
Ah yeah sorry my bad just noticed. Thanks!